lib/hypernova/version.rb

```diff
@@ -1,3 +1,3 @@
 module Hypernova
-  VERSION = "1.4.0"
+  VERSION = "2.0.0"
```

Comment:
please do revert this; versions are only changed in commits directly to master, not in PRs.

Reply:
Alright, I removed it 👍